src/main/drivers/light_led.h

```diff
@@ -48,6 +52,12 @@ PG_DECLARE(statusLedConfig_t, statusLedConfig);
 #define LED2_OFF                 NOOP
 #define LED2_ON                  NOOP
 
+#ifdef USE_RX_SPI
+#define RX_SPI_LED_TOGGLE        NOOP
```

Comment:
These should use function syntax to make the code easier to read. (i.e. they should be called as `rxSpiLedToggle()`.

Reply:
OK, I'll fix that.